packages/loot-core/src/server/budget/types/templates.d.ts

```diff
@@ -60,7 +60,7 @@ interface AverageTemplate extends BaseTemplate {
 }
 
 interface GoalTemplate extends BaseTemplate {
-  type: 'simple';
+  type: 'goal';
```

Comment:
Fixed since I'm pretty sure it's a typo, will matter later once we implement goal-type templates